src/style.css

```
@@ -1,3 +1,77 @@
+/* stylelint-disable rule-empty-line-before */
 body {
   margin: 0;
 }
+
+a {
+  color: #000;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
```

Default value, so you can just remove it
```suggestion
  font-weight: 500;
```